### PR TITLE
Drop underscore usage

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Even though underscore is already a dependency of backbone...

### CODE REVIEW

File: packages/controls/src/widget_selection.ts [hunk 1]
```diff
@@ -314,10 +313,7 @@ export class RadioButtonsView extends DescriptionView {
    */
   update(options?: any): void {
     const items: string[] = this.model.get('_options_labels');
-    const radios = _.pluck(
-      this.container.querySelectorAll('input[type="radio"]'),
-      'value'
-    );
+    const radios = this.container.querySelectorAll('input[type="radio"]').map(x => x.value);
```
Comment (jasongrout) [inline packages/controls/src/widget_selection.ts]: Does this work?

```suggestion
    const radios = Array.from(this.container.querySelectorAll('input[type="radio"]')).map(x => x.value);
```

File: packages/controls/src/widget_selection.ts [hunk 2]
```diff
@@ -502,7 +499,7 @@ export class ToggleButtonsView extends DescriptionView {
     const tooltips = this.model.get('tooltips') || [];
     const disabled = this.model.get('disabled');
     const buttons = this.buttongroup.querySelectorAll('button');
-    const values = _.pluck(buttons, 'value');
+    const values = buttons.map(x => x.value);
```
Comment (jasongrout) [inline packages/controls/src/widget_selection.ts]: Maybe this?
```suggestion
    const values = Array.from(buttons).map(x => x.value);
```

File: packages/controls/src/widget_audio.ts [hunk 3]
```diff
@@ -5,19 +5,18 @@ import { DOMWidgetView } from '@jupyter-widgets/base';
 
 import { CoreDOMWidgetModel } from './widget_core';
 
-import * as _ from 'underscore';
-
 export class AudioModel extends CoreDOMWidgetModel {
   defaults(): Backbone.ObjectHash {
-    return _.extend(super.defaults(), {
+    return {
+      ...super.defaults(),
```
Comment (jasongrout) [inline packages/controls/src/widget_audio.ts]: The disadvantage to this is that it creates a new object at every level of the class hierarchy, every time a class is created. The other would create one defaults dictionary at the root and modify it for each subclass.

However, perhaps that was just premature optimization and we shouldn't worry about it.

Comment (jasongrout) [reply packages/controls/src/widget_audio.ts]: If this really does become an issue, we can use `Object.assign` directly instead of underscore these days.

**Comment by jasongrout** [comment]
This works great; thanks!

**Comment by SylvainCorlay** [comment]
Thanks, I would have fixed it though :)

**Comment by jasongrout** [comment]
How about you review https://github.com/jupyter-widgets/ipywidgets/pull/2732 or https://github.com/jupyter-widgets/ipywidgets/pull/2532 instead? :)

**Pull request merged**